Lib/test/test_mmap.py

```diff
@@ -1058,6 +1058,44 @@ def __exit__(self, exc_type, exc_value, traceback):
                 with self.assertRaisesRegex(ValueError, "mmap closed or invalid"):
                     m.write_byte(X())
 
+    @unittest.skipUnless(os.name == 'nt', 'requires Windows')
+    def test_access_violations(self):
+        PAGE_NOACCESS = 0x01
+
+        with open(TESTFN, 'bw+') as f:
```

Comment:
A child process is needed in these tests because a regression could cause the process to crash. You should be able to use `assert_python_ok('-c', code)`, maybe with individual tests. Import it from `test.support.script_helper`.